.github/workflows/ci.yaml

```diff
@@ -138,7 +138,17 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        repository: [cmd-registry-memory, cmd-registry-proxy-dns, sdk-kernel, sdk-vppagent, cmd-nsmgr]
+        repository:
+          - sdk-vppagent
+          - sdk-kernel
+          - sdk-sriov
+          - cmd-nsmgr
+          - cmd-nsmgr-proxy
+          - cmd-registry-memory
+          - cmd-registry-proxy-dns
+          - cmd-forwarder-vppagent
+          - cmd-forwarder-sriov
```

Comment:
Note: sdk-sriov should update md-forwarder-sriov

Reply:
Fixed this and some more